app/lib/submission_builder/ty2024/states/id/documents/id40.rb

```diff
@@ -21,7 +21,7 @@ def document
                 add_non_zero_value(xml, :SpouseExemption, :ID40_LINE_6B)
                 add_non_zero_value(xml, :OtherExemption, :ID40_LINE_6C)
                 add_non_zero_value(xml, :TotalExemption, :ID40_LINE_6D)
-                unless @intake.filing_status_single? && @intake.dependents.count.zero?
+                unless @direct_file_data.primary_claim_as_dependent == "X"
```

Comment:
does this fill out even when filing status is `mfj`/`mfs`/`qw`/`hoh`? or is `filing_status_single?` still relevant?

Reply:
yes, i believe it should fill them out for all those statuses as well. got some clarification from margaret on this: https://codeforamerica.atlassian.net/browse/FYST-1794?focusedCommentId=15927